### PR TITLE
ui(homepage): improves text alignment

### LINKED ISSUE #2951: ui(homepage): Fix file name text alignment
In PR #2947, the heading was made to align in the centre which no doubt looks good. Unfortunately now the entire page is centre aligned. 

_Desktop_

Screenshot from 2019-04-19 15-55-52

_Mobile_

Screenshot from 2019-04-19 15-56-11

### Solution
The idea is to make the file names left aligned, why?
Because in all the webpack docs it left aligned, hence its the standard.

### PR DESCRIPTION
**Pull request by pranshuchittora**
Makes the file name to align left 
Closes #2951 

### Changes

_Desktop_

Screenshot from 2019-04-19 16-01-20

_Mobile_

Screenshot from 2019-04-19 16-01-31

### CODE REVIEW

File: src/components/Splash/Splash.scss [hunk 1]
```diff
@@ -64,6 +64,10 @@
     width: 100%;
     float: left;
 
+    p{
```
Comment (EugeneHlushko) [inline src/components/Splash/Splash.scss]:
```suggestion
    p {
```

File: src/components/Splash/Splash.scss [hunk 2]
```diff
@@ -64,6 +64,10 @@
...
+      text-align: start;
```
Comment (EugeneHlushko) [inline src/components/Splash/Splash.scss]:
```suggestion
      text-align: left;
```

**Comment by pranshuchittora** [comment]
https://github.com/webpack/webpack.js.org/pull/2952/commits/aa669bcecd749f2dbfa375918453d18d8df8f11b fixes inconsistent file formatting in the sass files.

@EugeneHlushko pls review the new changes. :+1:

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**